cros_gralloc/cros_gralloc_driver.cc

```diff
@@ -179,7 +184,7 @@ cros_gralloc_driver::cros_gralloc_driver()
 		switch (availabe_node) {
 		// only have one render node, is GVT-d/BM/VirtIO
 		case 1:
-			if (!drv_render_) {
+			if (drv_render_) {
```

Comment:
Seems we are altering the code functionality here, are we sure about this code change?